features/0510-dif-pres-exch-attach/README.md

```diff
@@ -241,7 +313,197 @@ Complete message example:
     }]
 }
 ```
+</details>
+
+
+### Supported Features of Presentation-Exchange
+
+Level of support for Presentation-Exchange features:
+
+Feature|Notes
+-------|-----
+`presentation_definition.input_descriptors.id`|
+`presentation_definition.input_descriptors.name`|
+`presentation_definition.input_descriptors.purpose`|
+`presentation_definition.input_descriptors.schema.uri`|URI for the credential's schema.
+`presentation_definition.input_descriptors.constraints.fields.path`|Array of JSONPath string expressions as defined in [section 8](https://identity.foundation/presentation-exchange/#jsonpath-syntax-definition). REQUIRED as per the spec.
+`presentation_definition.input_descriptors.constraints.fields.filter`|JSONSchema descriptor.
+`presentation_definition.input_descriptors.constraints.limit_disclosure`|`preferred` or `required` as defined in [the spec](https://identity.foundation/presentation-exchange/#input-descriptor-object) and as supported by the Holder and Verifier proof mechanisms.<br>Note that the Holder MUST have credentials with cryptographic proof suites that are capable of selective disclosure in order to respond to a request with `limit_disclosure: "required"`.<br/>See [RFC0593](../0593-json-ld-cred-attach/README.md) for appropriate crypto suites.
+`presentation_definition.input_descriptors.constraints.is_holder`|`preferred` or `required` as defined in [the spec](https://identity.foundation/presentation-exchange/#input-descriptor-object).<br>Note that this feature allows the Holder to present credentials with a different subject identifier than the DID used to establish the DIDComm connection with the Verifier.
+`presentation_definition.format`|For JSONLD-based credentials: `ldp_vc` and `ldp_vp`.
+`presentation_definition.format.proof_type`|For JSONLD-based credentials: `Ed25519Signature2018` and `BbsBlsSignature2020`.
```

Comment:
Some verifiers in the ecosystem will also support P Curves via `JsonWebSignature2020`.
Might be helpful to include this as a note.